packages/desktop/src/renderer/engine.ts

```diff
@@ -13,17 +13,17 @@ export const engine = new HedronEngine({
   onFrameStart: performanceMonitor.begin,
   onFrameEnd: performanceMonitor.end,
   rendererType: import.meta.env.HEDRON_RENDERER_TYPE ?? 'webgl',
+  clock,
 })
 
 export const engineStore = engine.getStore()
 
-export const useEngineStore = <T>(selector?: (state: EngineStateWithActions) => T) => {
-  return useStore(engineStore, selector!)
-}
 engine.registerPlugin(new MidiInput(engine))
+engine.registerPlugin(new LFOInput(engine))
```

Comment:
any thoughts on how to get these auto-picked up, wouldn't block it for this, I'm fond of decorators myself.

Also, wouldn't something like the lfo (and maybe midi in some curcumstances) be needed when running in a webpage, and should be handled in the engine instead of the desktop app (or maybe a third thing, plugin manager...)

Reply:
If you take a look at my web example PR (https://github.com/nudibranchrecords/hedron/pull/559), you still have to register the plugins you want. This is to keep things slim for web projects, only bringing in the parts you need.

For the desktop version we'll eventually want some extra layer where custom plugins can be automatically pulled in from the file system, etc.